packages/rendermime/src/outputmodel.ts

```diff
@@ -35,6 +40,11 @@ export interface IOutputModel extends IRenderMime.IMimeModel {
    */
   readonly executionCount: nbformat.ExecutionCount;
 
+  /**
+   * The observable data.
+   */
+  readonly observableText?: IObservableString;
```

Comment:
Should we maybe name it `streamedText` or `streamText` instead to avoid calling it by type name and highlight that it is only relevant to streams?